src/services/statisticMail/index.js

```diff
@@ -0,0 +1,44 @@
+const { Configuration } = require('@schul-cloud/commons');
+const hooks = require('./hooks/index');
+const globalHooks = require('../../hooks');
```

Comment:
Codacy found an issue: ['globalHooks' is assigned a value but never used.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045797)